images/peer/Dockerfile

```diff
@@ -20,19 +20,23 @@ RUN apk add --no-cache \
 	git \
 	make \
 	musl-dev
+RUN GO111MODULE=on go get github.com/mikefarah/yq/v4
 ADD . $GOPATH/src/github.com/hyperledger/fabric
 WORKDIR $GOPATH/src/github.com/hyperledger/fabric
 
 FROM golang as peer
 ARG GO_TAGS
 RUN make peer GO_TAGS=${GO_TAGS}
+RUN make ccaasbuilder
+RUN yq e -P '.chaincode.externalBuilders[0]={"name":"ccaas_builder","propagateEnvironment":["CHAINCODE_AS_A_SERVICE_BUILDER_CONFIG"],"path":"/opt/hyperledger/ccaas_builder"}' /go/src/github.com/hyperledger/fabric/sampleconfig/core.yaml > /go/src/github.com/hyperledger/fabric/sampleconfig/core_updated.yaml
```

Comment:
I think it would be better to just update sampleconfig/core.yaml, rather than inject the config here. Injecting it here incurs an extra yq dependency and makes the docker image build awkward. The sampleconfig/core.yaml is already intended to be used in the fabric peer docker image as-is.

Reply:
Yes that would be cleaner; I wasn't sure if updating the sample config would be ok.